<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="6" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="suite of suites" duration-ms="0" started-at="2020-07-15T14:49:57Z" finished-at="2020-07-15T14:49:57Z">
    <groups>
    </groups>
  </suite> <!-- suite of suites -->
  <suite name="Regression_Suite_stage_2" duration-ms="13" started-at="2020-07-15T14:49:56Z" finished-at="2020-07-15T14:49:56Z">
    <groups>
    </groups>
    <test name="Regression_Suite_stage_2" duration-ms="13" started-at="2020-07-15T14:49:56Z" finished-at="2020-07-15T14:49:56Z">
      <class name="com.selenium.testCases.GoogleLogout">
        <test-method status="FAIL" signature="clicklogoutButton()[pri:0, instance:com.selenium.testCases.GoogleLogout@5442a311]" name="clicklogoutButton" duration-ms="0" started-at="2020-07-15T14:49:56Z" finished-at="2020-07-15T14:49:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.selenium.testCases.GoogleLogout.clicklogoutButton(GoogleLogout.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clicklogoutButton -->
        <test-method status="PASS" signature="logoutButton()[pri:0, instance:com.selenium.testCases.GoogleLogout@5442a311]" name="logoutButton" duration-ms="1" started-at="2020-07-15T14:49:56Z" finished-at="2020-07-15T14:49:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutButton -->
        <test-method status="FAIL" signature="verifylogoutButton()[pri:0, instance:com.selenium.testCases.GoogleLogout@5442a311]" name="verifylogoutButton" duration-ms="1" started-at="2020-07-15T14:49:56Z" finished-at="2020-07-15T14:49:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.selenium.testCases.GoogleLogout.verifylogoutButton(GoogleLogout.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifylogoutButton -->
      </class> <!-- com.selenium.testCases.GoogleLogout -->
    </test> <!-- Regression_Suite_stage_2 -->
  </suite> <!-- Regression_Suite_stage_2 -->
  <suite name="Regression_Suite_stage_1" duration-ms="62" started-at="2020-07-15T14:49:53Z" finished-at="2020-07-15T14:49:53Z">
    <groups>
    </groups>
    <test name="Regression_Suite_stage_1" duration-ms="62" started-at="2020-07-15T14:49:53Z" finished-at="2020-07-15T14:49:53Z">
      <class name="com.selenium.testCases.GoogleLogin">
        <test-method status="PASS" signature="googleAdmin()[pri:0, instance:com.selenium.testCases.GoogleLogin@46d56d67]" name="googleAdmin" duration-ms="12" started-at="2020-07-15T14:49:53Z" finished-at="2020-07-15T14:49:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleAdmin -->
        <test-method status="PASS" signature="googleLogin()[pri:0, instance:com.selenium.testCases.GoogleLogin@46d56d67]" name="googleLogin" duration-ms="8" started-at="2020-07-15T14:49:53Z" finished-at="2020-07-15T14:49:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleLogin -->
        <test-method status="FAIL" signature="googleProfile()[pri:0, instance:com.selenium.testCases.GoogleLogin@46d56d67]" name="googleProfile" duration-ms="0" started-at="2020-07-15T14:49:53Z" finished-at="2020-07-15T14:49:53Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.selenium.testCases.GoogleLogin.googleProfile(GoogleLogin.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1204)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- googleProfile -->
      </class> <!-- com.selenium.testCases.GoogleLogin -->
    </test> <!-- Regression_Suite_stage_1 -->
  </suite> <!-- Regression_Suite_stage_1 -->
</testng-results>
